src/generator.cs

```diff
@@ -7081,5 +7103,12 @@ string RenderType (Type t)
 			return t.FullName;
 		
 	}
-	
+
+	public static string Quote (string s)
+	{
+		if (String.IsNullOrEmpty (s))
+			return s ?? String.Empty;
```

Comment:
If `s` is an empty string, this will return an empty string (not a quoted empty string).

Example:

API definition has this:

```
[NotImplemented ("")]
```

and this will print:

```
[NotImplemented()]
```